modules/installation-vsphere-machines.adoc

```diff
@@ -7,30 +7,26 @@
 [id="installation-vsphere-machines_{context}"]
 = Creating {op-system-first} machines in vSphere
 
-Before you install a cluster that contains user-provisioned infrastructure on
-VMware vSphere, you must create {op-system} machines on vSphere hosts for it to
-use.
+Before you install a cluster that contains user-provisioned infrastructure on VMware vSphere, you must create {op-system} machines on vSphere hosts for it to use.
 
 .Prerequisites
 
 * Obtain the Ignition config files for your cluster.
-* Have access to an HTTP server that you can access from your computer and that
-the machines that you create can access.
+* Have access to an HTTP server that you can access from your computer and that the machines that you create can access.
 * Create a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
 
 .Procedure
 
-. Upload the bootstrap Ignition config file, which is named
-`<installation_directory>/bootstrap.ign`, that the installation program created
-to your HTTP server. Note the URL of this file.
-+
-You must host the bootstrap Ignition config file because it is too large to
-fit in a vApp property.
```

Comment:
@dav1x, will you PTAL?

Reply:
@kalexand-rh You can store the bootstrap ignition in the VM's extra config properties. This eliminates the need for an external web server. 

https://github.com/openshift/installer/blob/master/upi/vsphere/vm/main.tf#L32-L33

But, it may add some complexity to the documentation.

Reply:
Thanks, @dav1x - IIUC, this simplifies the docs (at least here) because we can remove these first steps about uploading and hosting bootstrap Ignition config (as these steps are now commented out). 

Or do we need to add instructions about storing in VM's config properties? CC: @jcpowermac